src/app/guides/page.tsx

```diff
@@ -667,6 +674,34 @@ const StyleGuide: React.FC = () => {
           onChange={handleTextChange}
         />
       </div>
+      <div className="w-full">
+        <h2 className="mb-3 text-2xl font-semibold text-gray-800">
+          Email Template Paginated Table List
+        </h2>
+
+        <div className="rounded-lg bg-zinc-950 p-4">
+          <span className="block font-mono text-sm text-gray-100">
+            <span className="text-pink-400">PaginatedTemplateList</span> {"{"}
+          </span>
+          <span className="ml-4 block font-mono text-sm text-gray-100">
+            itemsPerPage<span className="text-pink-400">: </span>number;
+            <br />
+            onPreview<span className="text-pink-400">: </span>() =&gt; void;
+            <br />
+            templates: [];
+            <br />
+            activePreview<span className="text-pink-400">: </span>number;
+            <br />
+          </span>
+          <span className="block font-mono text-sm text-gray-100">{"}"}</span>
```

Comment:
What is happening here with the squigly brackets?

Reply:
I am trying to show the types required for the usage of the component. Used an approach from what we previously have in the file